core/commands/files/files.go

```diff
@@ -61,7 +63,24 @@ var FilesStatCmd = &cmds.Command{
 	Arguments: []cmds.Argument{
 		cmds.StringArg("path", true, false, "Path to node to stat."),
 	},
+	Options: []cmds.Option{
+		cmds.StringOption("format", "Print statistics in given format. Allowed tokens: "+
+			"<hash> <size> <cumulsize> <type> <childs>. Conflicts with other format options.").Default(
+			`<hash>
```

Comment:
This should be prefixed with `Hash: <hash>` -- otherwise it only prints the hash in one line and then `Key: Value` format.